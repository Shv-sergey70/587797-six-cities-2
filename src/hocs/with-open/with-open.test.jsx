import React from 'react';
import renderer from 'react-test-renderer';
import WrappedComponent from './test-component';

it(`HoC with-open correctly renders`, () => {
  const tree = renderer.create(
      <WrappedComponent/>).toJSON();

  expect(tree).toMatchSnapshot();
});
