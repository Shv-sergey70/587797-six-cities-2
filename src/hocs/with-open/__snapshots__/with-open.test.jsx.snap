// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HoC with-open correctly renders 1`] = `
<div
  onClick={[Function]}
>
  TEST
</div>
`;
